### DISCUSSION TITLE
"quarto-permission-test" empty confluence pages dumped to root confluence space

### DISCUSSION THREAD

**Discussion by angelotrivelli** [quarto-dev/quarto-cli#5295] [Q&A]
I found that when I tried to make quarto publish to a particular confluence space, it put empty pages in the root of that space. All the pages were titled like...

      `quarto-permission-test-71dc05b5-664b-4ac5-a0de-673c51bdbc03`

Where the `71dc...` is obviously some GUID, different for every page. 

For whatever reason, I am not able to publish an actual page to that confluence space. Instead it just dumps one of those empty files into the root of the space. 

If I publish to my personal space, I found I can then manually move my page from the personal space into wherever I want it, but I can't do it directly from quarto publish.

Unfortunately, I am not super proficient in confluence, nor quarto. Are there any pointers for troubleshooting what's going here? It's kind of embarrassing that quarto dumped a empty page like that (with my name on it) into a highly visible Confluence space.

**Comment by ghost** [comment]
Thanks for posting this.  I've moved this discussion to this issue - let's continue working through it there:

https://github.com/quarto-dev/quarto-cli/issues/5299
